wen-restart/src/wen_restart.rs

```diff
@@ -405,6 +419,210 @@ fn find_bankhash_of_heaviest_fork(
     Ok(parent_bank.hash())
 }
 
+// Aggregate the heaviest fork and send updates to the cluster.
+pub(crate) fn aggregate_restart_heaviest_fork(
+    wen_restart_path: &PathBuf,
+    wait_for_supermajority_threshold_percent: u64,
```

Comment:
Seems better here just to pass in the `wait_for_supermajority_threshold_percent - 5%` number, that way we don't have to run the calculation multiple times within the function

Reply:
Other than tests, this function is only called from the main function `wait_for_wen_restart()`. And this function should only be called once unless the validator is restarted.

I prefer to avoid putting HeaviestFork specific logic into the main function, but I can move the logic to the main function if you feel strongly about it.

Reply:
it's fine as is 👍